src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import core.basesyntax.service.ColorSupplier;
+
+public abstract class Figure implements CalculatorArea, Drawable {
+    protected static final int MAX_VALUE = 100;
```

Comment:
This constant must be in the FigureSuplier class, as must the logic for creating all random shapes